lib/ui/platform_dispatcher.dart

```diff
@@ -168,6 +168,35 @@ class PlatformDispatcher {
   // A map of opaque platform view identifiers to view configurations.
   final Map<Object, ViewConfiguration> _viewConfigurations = <Object, ViewConfiguration>{};
 
+  /// The implicit [FlutterView] provided by the platform, if any.
+  ///
+  /// If the platform provides an implicit view, it can be used to bootstrap
+  /// the framework. This is common for platforms designed for single-view
+  /// applications (e.g. on mobile devices with a single display).
+  ///
+  /// Applications and libraries must not rely on this property being set
+  /// as it may be null depending on the engine's configuration. Instead,
+  /// consider using [View.of] to lookup the [FlutterView] the current
+  /// [BuildContext] is drawing into.
```

Comment:
Overall this is good. Rather than pointing out that it may be null (clear from the method signature already), I'd explain _when_ this will be null or non-null, and follow up with your existing suggestion to avoid this in favour of `View.of`.

Reply:
> I'd explain _when_ this will be null or non-null

It's hard to explain when this is null as it is rather arbitrary from the perspective of the engine/framework.

The current plan is:

1. Mobile apps will be non-null
2. Desktop apps using today's runner will be non-null, even if they have multiple windows. This ensures backwards compatibility
3. Desktop apps using a future runner that doesn't exist yet will be null

However, a brand new desktop embedder could choose to never provide an implicit view. In other words, whether there is an implicit view depends on your embedder & your runner app. 

Let me think on this some more...

Reply:
I tried to clarify this, see: https://github.com/flutter/engine/pull/39553#discussion_r1107582717